src/main/java/seedu/address/storage/JsonAdaptedPolicy.java

```diff
@@ -0,0 +1,152 @@
+package seedu.address.storage;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.policy.Coverage;
+import seedu.address.model.policy.Description;
+import seedu.address.model.policy.EndAge;
+import seedu.address.model.policy.Policy;
+import seedu.address.model.policy.PolicyName;
+import seedu.address.model.policy.Price;
+import seedu.address.model.policy.StartAge;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Jackson-friendly version of {@link seedu.address.model.person.Person}.
+ */
+class JsonAdaptedPolicy {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Policy's %s field is missing!";
+
+    private final String name;
+    private final String description;
+    private final String coverage;
+    private final String price;
+    private final String startAge;
+    private final String endAge;
+    private final List<JsonAdaptedTag> criteria = new ArrayList<>();
+    private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+
+    /**
+     * Constructs a {@code JsonAdaptedPolicy} with the given policy details.
+     */
+    @JsonCreator
+    public JsonAdaptedPolicy(@JsonProperty("name") String name, @JsonProperty("description") String description,
+                             @JsonProperty("coverage") String coverage, @JsonProperty("price") String price,
+                             @JsonProperty("start age") String startAge, @JsonProperty("end age") String endAge,
+                             @JsonProperty("criteria") List<JsonAdaptedTag> criteria,
+                             @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+        this.name = name;
+        this.description = description;
+        this.coverage = coverage;
```

Comment:
![Screenshot 2019-10-07 at 2 10 00 AM](https://user-images.githubusercontent.com/40198156/66273673-78c77c80-e8a8-11e9-857a-7091263289c3.png)
Yo i'm trying to fix the test cases.

Test case:`toModelType_validPersonDetails_returnsPerson`
In `SampleDataUtil`, `new Coverage(" days/10 months/11 years/12")`
However in the model, coverage takes in the argument `10 days 11 months 12 years`, which leads to coverage being null as it is not able to parse the correct format using `getCoverageBreakdown`. 

However, is coverage meant to be doing the parsing? cuz in sample data util, the arguments are the content itself.